src/components/Spinner/Spinner.js

```diff
@@ -14,11 +14,10 @@ export const Spinner = (props) => {
   ];
 
   return (
-    <div className="slds-spinner_container">
-      <div {...rest} className={cx(sldsClasses)} aria-hidden="false" role="alert">
-        <div className="slds-spinner__dot-a" />
-        <div className="slds-spinner__dot-b" />
-      </div>
+    <div {...rest} className={cx(sldsClasses)} role="status">
```

Comment:
Why not `<div className={cx(sldsClasses)} role="status" {...rest}>`?

That way the user could customize certain props. /cc @fspoettel